mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/service/impl/CourseNotificationServiceImpl.java

```diff
@@ -188,26 +187,36 @@ public void updateSmsStatus(MotechEvent event) {
     private String buildSmsContent(Long callingNumber) {
 
         FrontLineWorker flw = frontLineWorkerService.getByContactNumber(callingNumber);
+        String locationId = "XX"; // unknown location id
+        String smsLanguage = null;
+
         if (flw == null) {
             throw new IllegalStateException("Unable to find flw for calling number: " + callingNumber);
         }
-        Language flwLanguage = flw.getLanguage() != null ? flw.getLanguage() : flw.getDistrict().getLanguage();
-        String smsLanguage;
-        if (flwLanguage != null && flwLanguage.getCode() != null) {
-            smsLanguage = flwLanguage.getCode();
+
+        // Build location code
```

Comment:
it is location identifier string that needs to be built.. doesn't matter what we call it here. Open to whatever sounds good.